doc/yaml/sql/mob_db.sql

```diff
@@ -63,6 +63,12 @@ CREATE TABLE `mob_db` (
   `racegroup_illusion_turtle` tinyint(1) unsigned DEFAULT NULL,
   `racegroup_rachel_sanctuary` tinyint(1) unsigned DEFAULT NULL,
   `racegroup_illusion_luanda` tinyint(1) unsigned DEFAULT NULL,
+  `racegroup_illusion_frozen` tinyint(1) unsigned DEFAULT NULL,
+  `racegroup_illusion_moonlight` tinyint(1) unsigned DEFAULT NULL,
+  `racegroup_ep16_def` tinyint(1) unsigned DEFAULT NULL,
+  `racegroup_edda_arunafeltz` tinyint(1) unsigned DEFAULT NULL,
+  `racegroup_lasagna` tinyint(1) unsigned DEFAULT NULL,
+  `racegroup_glast_heim_abyss` tinyint(1) unsigned DEFAULT NULL,
```

Comment:
Could we detect these issues with CI somehow?

Reply:
these should probably be in a new table, just like drops
`monsterid|racegroup|enabled` (enabled is true by default, only set to false if the import removes it)

Reply:
How is that related to my comment?

But, yes, they should be. But the whole SQL support is only there for backwards compatibility with flux and other CPs.

Reply:
hmm anyway I skip that, the check/revamp should not be done in this PR